### DISCUSSION TITLE
Does the Developer-Guide still exist?

### DISCUSSION THREAD

**Discussion by LPlanke** [leihs/leihs#1694] [Q&A / Community Support]
The [Contributing](https://github.com/leihs/leihs/wiki/Contributing) wiki page mentions a developer guide several times, but the page doesn't seem to exist. Has it been lost? Is there a way to access an old version of it? If so, would the content still be helpful? If not, would it be possible to create a new, up to date version of it?

**Comment by nimaai** [comment]
No. All the relevant developer information has been split and moved to the READMEs of the respective submodule repos. I have corrected the contribution page. Thanks for the hint.

**Comment by nimaai** [comment]
The READMEs should be quite up-to-date. If you discover something outdated and not working (anymore) kindly let us know or make a PR.
